bff/pkg/server/server.go

```diff
@@ -79,7 +99,7 @@ func (s server_imp) Run() error {
 
 	r.Use(CORS)
 
-	r.Get("/api/v1/activity", activity_handler(s.metrics_service))
+	r.Get("/api/v1/activity", activity_handler(s.metrics_service, s.validate))
 	r.Get("/api/v1/instance", info_handler(s.metrics_service, s.dbIdentifiers))
 	r.Get("/api/v1/instance/{dbIdentifier}/database", databases_handler(s.metrics_service))
```

Comment:
Don't we need `s.validate` passed to `info_handler` and `databases_handler`?

Reply:
are we adding validation for these routes in this PR ? The database route in particular doesnt actually use the dbIdentifer value it accepts. We can add validation for that when we actually use it